app/services/orderService.js

```diff
@@ -82,9 +80,69 @@ class OrderService {
         strategy.now_buy_times = now_buy_times;
         await strategy.save();
 
-        return { order_id: order_id }
+        return { order_id: inOrder.order_id }
     }
 
+    async sellOrder(strategy_id) {
+        const item = await Strategy.findById(strategy_id);
+        // Access the current price
+        const exchange = new (ccxt)[item.exchange]({
+            apiKey: item.key,
+            secret: item.secret,
+            timeout: 5000,
+        });
+        
+        const tickerRes =  await exchange.fetchTicker(item.symbol);
```

Comment:
Two different naming style, please use one consistently.

tickerRes of camel / sell_price of snake?